build.sbt

```diff
@@ -58,22 +51,19 @@ libraryDependencies ++= Seq(
   jdbc,
   ehcache,
   ws,
-  "com.typesafe.akka" %% "akka-agent" % "2.5.32",
-  "io.reactivex" %% "rxscala" % "0.27.0",
   "com.amazonaws" % "aws-java-sdk-cloudwatch" % awsSdkVersion,
   "com.amazonaws" % "aws-java-sdk-ses" % awsSdkVersion,
   "com.google.code.gson" % "gson" % "2.10.1",
-  "com.gu.play-googleauth" % "play-v28_2.13" % "2.4.0",
+  "com.gu.play-googleauth" %% "play-v30" % "3.0.6",
```

Comment:
Great fix. I'd love to know the story behind why this line was the way it was! 🤪